transliterator.py

```diff
@@ -59,15 +59,17 @@ def main():
 	parser.add_argument("--subscription_key", default=None, type=str, required=True, help="Azure Subscription key for downloading transliterations")
 	parser.add_argument("--input_file", default=None, type=str, required=True,
 						help="The roman hindi words vocabulary ")
+	parser.add_argument("--subscription_region", default="southeastasia", type=str, required=True, help="Azure Subscription Region")
 	
 	args = parser.parse_args()
 	input_file = args.input_file
 	subscription_key = args.subscription_key
 
-	req_token = get_token(subscription_key)
+	req_token = get_token(subscription_key, args.subscription_region)
 	headers = { 'Accept': 'application/json;text/xml',
 			'Content-Type': 'application/json',
 			'Ocp-Apim-Subscription-Key': subscription_key,
+			'Ocp-Apim-Subscription-Region': args.subscription_region,
```

Comment:
To add, I'm not able to get your branch to work. Removing this line makes it work

Reply:
Strange, if I remove this, then code does not work. I get this error
```
{'error': {'code': 401000, 'message': 'The request is not authorized because credentials are missing or invalid.'}}
```